inttest/cover/cover_rt.erl

```diff
@@ -0,0 +1,50 @@
+-module(cover_rt).
+
+-export([files/0,run/1]).
+
+-include_lib("eunit/include/eunit.hrl").
+
+files() ->
+    [{create, "ebin/foo.app", app(foo)},
+     {copy, "../../rebar","rebar"},
+     {copy, "src", "src"},
+     {copy,
+      "rebar-cover_compile_opts.config",
+      "rebar-cover_compile_opts.config"}].
+
+run(Dir) ->
+    io:format("run(~p)~n",[Dir]),
```

Comment:
Dave built retest for rebar integration tests, and there is no documentation besides the tests, but for whats it's worth it's very simple.